.gitignore

```diff
@@ -65,3 +65,5 @@ About
 
 /venv/
 /.idea/
+log/
+install/
```

Comment:
Could you move these to the top-most "section"? That keeps the .gitignore (kind of) organised